build-scripts/runtime_lib.cmake

```diff
@@ -165,7 +165,11 @@ file (GLOB header
 )
 LIST (APPEND RUNTIME_LIB_HEADER_LIST ${header})
 
-enable_language (ASM)
+if (MSVC)
```

Comment:
Had better use `if (WAMR_BUILD_PLATFORM STREQUAL "windows")` to control the code like the original code in samples?  Not sure whether other windows compiler like `MINGW` also needs to enable ASM_MASM.